src/model-cache.ts

```diff
@@ -109,30 +109,29 @@ export default class ModelCache implements IModelCache {
         const results = await generator();
 
         function getData(data: any) {
-            if (results === null) {
+            if (data === null) {
                 return 'null'; // JSON stringified "null"
             }
 
             if (options.raw || options.plain) {
                 return JSON.stringify(data);
             }
 
-            if (!results) {
-                return results;
+            if (typeof data === 'number') {
+                // like for count() calls
+                return data.toString();
             }
 
-            if (typeof results === 'number') {
-                // like for count() calls
-                return results.toString();
+            if (!data) {
```

Comment:
i am leaving this as is just for now and will address in a follow up PR based on our discussion in #typescript channel. The reason is this was the behavior Jah expected here in his code and here `falsyness` might mean multiple things so want to make a more careful call here

Reply:
Neverr mind fixed it as i see no other cahse